client/MentorPanel/src/components/questions.js

```diff
@@ -19,7 +19,9 @@ const Questions = () => {
   }
 
   const questions = mentor.topic_questions[current_topic]
+  const recommended = mentor.topic_questions['Recommended'] ? mentor.topic_questions['Recommended'] : []
```

Comment:
this can be simplied, no?

```
const recommended = mentor.topic_questions['Recommended'] || []
```

I guess if you wanted to be strict that the value is an array it would be the below, but seems like that value would ALWAYS be an array or undefined, no?

```
const recommended = Array.isArray(mentor.topic_questions['Recommended'] )? mentor.topic_questions['Recommended'] : []
```